_build-and-test-all.sh

```diff
@@ -32,4 +33,18 @@ echo 'show dbs' |  ./mongodb-cli.sh -i
 
 ./gradlew :end-to-end-tests:cleanTest :end-to-end-tests:test
 
+if [ "${DATABASE}" == "mysql" ]; then
+  cat "add-database-id-support-to-eventuate-mysql.migration.sql" | ./mysql-cli.sh -i
+fi
+
+manual_compose="docker-compose -f docker-compose-mysql-binlog.yml"
+services_to_restart="order-service customer-service order-history-service"
+
+${manual_compose} stop ${services_to_restart}
```

Comment:
Shouldn't the CDC be stopped too? It needs to be configured with the outbox ID

Reply:
Current implementation requires to specify EVENTUATE_OUTBOX_ID immediately after upgrade even if it is not used.